extensions/azure-storage-blob/runtime/src/main/java/io/quarkiverse/azure/storage/blob/runtime/StorageBlobServiceClientProducer.java

```diff
@@ -3,16 +3,22 @@
 import jakarta.enterprise.inject.Produces;
 import jakarta.inject.Inject;
 
+import com.azure.core.util.ClientOptions;
 import com.azure.storage.blob.BlobServiceClient;
 import com.azure.storage.blob.BlobServiceClientBuilder;
 
 public class StorageBlobServiceClientProducer {
 
+    private static final String AZURE_QUARKUS_STORAGE_BLOB = "az-qk-storage-blob";
```

Comment:
Have you tested it?
What's the UserAgent looks like, could you give an example here?

Reply:
See [User Agent Format](https://azure.github.io/azure-sdk/general_azurecore.html#telemetry-policy)

Reply:
With the help from @backwind1233, I followed [Enable Logging](https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/core/azure-core/README.md#enabling-logging) and verified that it works:

```
export AZURE_LOG_LEVEL=VERBOSE
export AZURE_HTTP_LOG_DETAIL_LEVEL=headers

# Switch to in integration-tests/azure-app-configuration and integration-tests/azure-storage-blob,
# follow readme to build and run the sample app in JVM mode
java -jar ./target/quarkus-app/quarkus-run.jar
```

I observed the application id is displayed in field "User-Agent":
* `az-qk-app-config` for azure-app-configuration:
  ```
  {"az.sdk.message":"HTTP request","method":"GET","url":"https://jiangnma072723appconfig.azconfig.io/kv?api-version=1.0","tryCount":"1","Date":"Thu, 27 Jul 2023 08:28:49 GMT","Authorization":"REDACTED","Content-Length":"0","x-ms-return-client-request-id":"true","x-ms-content-sha256":"REDACTED","Sync-Token":"REDACTED","Host":"REDACTED","Content-Type":"application/json","x-ms-client-request-id":"e9a7c15e-d418-4392-aecb-04536492c6ad","User-Agent":"az-qk-app-config azsdk-java-azure-data-appconfiguration/1.4.6 (17.0.7; Linux; 5.10.102.1-microsoft-standard-WSL2)","Accept":"application/vnd.microsoft.azconfig.kv+json","contentLength":0}
  ```
 
*  `az-qk-storage-blob` for azure-storage-blob after issuing a cURL command to create a blob `curl http://localhost:8080/quarkus-azure-storage-blob/testcontainer/testblob -X POST -d 'Hello Quarkus Azure Storage Blob!' -H "Content-Type: text/plain"`:
   ```
   {"az.sdk.message":"HTTP request","method":"PUT","url":"https://jiangma072723sa.blob.core.windows.net/testcontainer?restype=container","tryCount":"1","Date":"Thu, 27 Jul 2023 08:51:18 GMT","Authorization":"REDACTED","Content-Length":"0","x-ms-version":"2022-11-02","x-ms-client-request-id":"0d73fb7a-be32-4e6c-a2f9-5fcaf7bc20fe","Accept":"application/xml","User-Agent":"az-qk-storage-blob azsdk-java-azure-storage-blob/12.22.3 (17.0.7; Linux; 5.10.102.1-microsoft-standard-WSL2)","contentLength":0}
   ```